src/main/scala/wdl4s/types/WdlType.scala

```diff
@@ -28,8 +29,13 @@ trait WdlType {
    */
   def coerceRawValue(any: Any): Try[WdlValue] = {
     any match {
-      case v: WdlValue if v.wdlType == this => Success(v)
-      case a if !coercion.isDefinedAt(any) => Failure(new IllegalArgumentException(s"No coercion defined from '$any' of type '${any.getClass}' to ${getClass.getSimpleName}."))
+      case wdlValue: WdlValue if wdlValue.wdlType == this => Success(wdlValue)
+      case wdlValue: WdlValue if !coercion.isDefinedAt(any) => Failure(new IllegalArgumentException(
+        s"No coercion defined from '${wdlValue.toWdlString}' of type" +
```

Comment:
I personally don't like including the value here. It's almost always unreadably long and not very useful in debugging...